src/main.ts

```diff
@@ -1,9 +1,12 @@
-import config from "./config.json" assert { type: "json" };
+import configuration from "./config.json" assert { type: "json" };
```

Comment:
question: why? `config` and `configCommand` do not collide.

suggestion: maybe we call commands with `Cmd` suffix? `configCmd` is shorted, still self-explanatory, and doesn't collide with the development configuration file.

Reply:
It was renamed when config was called as a console's command config. So I decided to rename it during development. For now it can be returned back. Yes, we can use shorten form with 'dev' suffix